packages/kit/src/utils/filesystem.js

```diff
@@ -41,8 +41,11 @@ export function copy(from, to, filter = () => true) {
 	return files;
 }
 
-/** @param {string} cwd */
-export function walk(cwd) {
+/**
+ * @param {string} cwd
+ * @param {boolean} [dirs]
```

Comment:
```suggestion
 * @param {boolean} dirs
```
Maybe also add a little comment what this param does? In general maybe a little doc on what is returned by this function. Or better yet, adjust the name to better tell what it actually does, like `collect_filenames`